index.js

```diff
@@ -256,6 +256,29 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options = {}) => {
+		const killResult = originalKill(signal);
+		if (signal === undefined ||
+			(typeof signal === 'string' && signal.toUpperCase() === 'SIGTERM') ||
+			(typeof signal === 'number' && signal === os.constants.signals.SIGTERM)) {
```

Comment:
```suggestion
		signal === os.constants.signals.SIGTERM) {
```

The `typeof signal === 'number'` is not needed since you're comparing with a `number` afterwards.

Reply:
Good point, fixed: 2987b9fbe8cf34d560942360a2c5e095f0b83108